README.md

```diff
@@ -57,9 +60,56 @@ Google APIs Client Libraries, in [Client Libraries Explained][explained].
 
 ### Installing the client library
 
-    npm install --save text-to-speech
+    npm install --save @google-cloud/text-to-speech
 
+### Using the client library
 
+```javascript
+const fs = require('fs');
+
+// Imports the Google Cloud client library
+const textToSpeech = require('@google-cloud/text-to-speech');
+
+// Creates a client
+const client = new textToSpeech.TextToSpeechClient();
+
+// The text to synthesize
+const text = 'Hello, world!';
+
+// Construct the request
+const request = {
+  input: {text: text},
+  // Select the language and SSML Voice Gender (optional)
+  voice: {languageCode: 'en-US', ssmlGender: 'NEUTRAL'},
+  // Select the type of audio encoding
+  audioConfig: {audioEncoding: 'MP3'},
+};
+
+// Performs the Text-to-Speech request
+client
+  .synthesizeSpeech(request)
+  .then(results => {
+    // The binary audio content returned from the API
+    const audioContent = results[0].audioContent;
+
+    // Write the audio content to a local file
+    fs.writeFileSync('output.mp3', audioContent, 'binary');
```

Comment:
since we're already in `.then()`, can we change it to async version? Users might copy this code and using `writeFileSync` is not a good way of working with files, especially large `.mp3` files. I would suggest using `util.promisify` or `pify` package and convert this and other calls to synchronous `fs` methods to promises.

Reply:
This code is in a SNIPPET on the website and therefore it should have zero dependencies other than `@google-cloud/text-to-speech`, if possible.

It's common for snippets to pull in their relative System.IO SDK for filesystem reads/writes, but everything should be as native and simple as possible.

Snippets should NEVER use anything besides:

 1. the cloud client library being imported
 2. required SDK functionality, built into the language's SDK

Usually only tutorials break this rule, eg. if you want to draw boxes on images.... you're gonna want some imagemagick! 😄 

----

If this is doable with native fs and async/await, then +1 presuming it doesn't ugly the code at all!

Reply:
*updating...*

Reply:
done.